jlexer/bytestostr_nounsafe.go

```diff
@@ -0,0 +1,10 @@
+//+build easyjson_nounsafe appengine
```

Comment:
Thanks for the feedback. Added more documentation as requested:
https://github.com/mailru/easyjson/pull/104/commits/5d608ccfff4bf40c244150ff479c865779073608